spell/psm-checklists.md

```diff
@@ -0,0 +1,113 @@
+    - `LitePsmJob`
+        - [ ] Uses the default Solidity version in [`dss-cron`](https://github.com/makerdao/dss-cron)
+        - [ ] Optimizer is **enabled**
+        - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-cron)
+        - Constructor params:
+            - [ ] `_sequencer` matches `CRON_SEQUENCER` address from the chainlog
+            - [ ] `_litePsm` matches `DssLitePsm` address
+            - [ ] `_rushThreshold` matches the Exec Sheet (it might be named as "fill threshold")
+            - [ ] `_gushThreshold` matches the Exec Sheet (it might be named as "trim threshold")
+            - [ ] `_cutThreshold` matches the Exec Sheet (it might be named as "chug threshold")
+- Initialization
+    - `DssLitePsm`:
+        - [ ] `vow` is set to `MCD_VOW` from the chainlog
+        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike(LITE_PSM).kiss(MCD_PAUSE_PROXY)`)
+        - [ ] `DssLitePsmMom` is authed (i.e. `RelyLike(LITE_PSM).rely(MOM)`)
+    - `DssLitePsmMom`:
+        - [ ] The chief (`MCD_ADM`) is set as the `authority` (i.e. `MomLike(MOM).setAuthority(MCD_ADM)`)
+    - `LitePsmJob`:
+        - [ ] Job is added to `CRON_SEQUENCER`
+    - `MCD_SPOT`:
+        - [ ] Dai parity (`par`) current value is 1 (`1 * RAY`)
+        - [ ] Collateralization ratio (`mat`) is set to 100% (`1 * WAD`) for the ilk 
+        - IF a new `pip` is being used:
+            - [ ] `pip` is set for the ilk
+            - [ ] `pip` is a `DSValue` instance
+            - [ ] The value on `pip` is set to 1 (`1 * WAD`)
+            - [ ] `pip` deployer is no longer the `owner`
+            - [ ] `MCD_PAUSE_PROXY` is the `owner` on `pip`
+        - OTHERWISE when reusing an existing `pip`:
+            - [ ] `pip` is set for the ilk
+            - [ ] `pip` in the chainlog matches `gem` symbol (i.e. `PIP_USDC` for `USDC`)
+    - `MCD_VAT`:
+        - [ ] New ilk is initialized
+        - [ ] `urns[ilk][LITE_PSM].ink` is set to the max value that will not cause an overflow (`int256(type(uint256).max / RAY)`)
+    - `MCD_JUG`:
+        - [ ] New ilk is initialized
+        - [ ] Stability fee (`duty`) is set to 0% (`1 * RAY`) for the ilk
+    - `CHAINLOG`:
+        - [ ] `DssLitePsm` is added to the chainlog according to the Exec Sheet
+        - [ ] `pocket` is added to the chainlog according to the Exec Sheet
+        - [ ] `DssLitePsmMom` is added to the chainlog according to the Exec Sheet
+        - [ ] `LitePsmJob` is added to the chainlog according to the Exec Sheet
+        - IF a new `pip` is being added
+            - [ ] `pip` is added to the chainlog as `PIP_{TOKEN_SYMBOL}` (i.e. `PIP_USDC`)
+        - [ ] Version is bumped: `{x}.{y}.{z+1}`
+    - `ILK_REGISTRY`:
+        - [ ] New ilk is added to the registry
+            - [ ] The new class for join-less ilks is used (`REG_CLASS = 6`)
```

Comment:
Do we also need some base tests for this or do you plan to modify existing ones?
Anyway, this checklist is currently missing test coverage.

Reply:
There is no general rule for ilk class in the base tests. Not sure if we should add this, up for discussion.
In the meantime, I'll put this as an item for test coverage, which is missing.